src/azure-cli/azure/cli/command_modules/advisor/custom.py

```diff
@@ -132,7 +131,7 @@ def _parse_recommendation_uri(recommendation_uri):
 def _generate_recommendations(client):
     from msrestazure.azure_exceptions import CloudError
 
-    response = sdk_no_wait(True, client.generate)
+    response = client.generate(raw=True)
```

Comment:
Maybe we can split the migration of **advisor** and **backup** to a different PR?

Reply:
I can split it to another PR.

---
In reply to: [406657692](https://github.com/Azure/azure-cli/pull/12952#discussion_r406657692) [](ancestors = 406657692)